modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -4,12 +4,12 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-In Anypoint Exchange private and Exchange portal, to share an API, example, template, or connector means that
+In Anypoint Exchange private and Exchange portal, to share an API, policy, example, template, or connector means that
 you first create an asset corresponding to the asset type.
 
 How you create an asset depends on its type.
 
-* For OAS 2.0, RAML, RAML fragments, HTTP, WSDL, and Custom assets, create the asset directly using the Exchange *New asset* menu.
+* For OAS 2.0, RAML, RAML fragments, HTTP, WSDL, Policy, and Custom assets, create the asset directly using the Exchange *New asset* menu.
```

Comment:
Please remove this, policies can only be created by Exchange Maven Facade. Should we clarify this somewhere?

Reply:
It's here https://docs.mulesoft.com/exchange/to-publish-assets-maven maybe it's enough. We can add the link, as the connectors: https://github.com/mulesoft/docs-exchange/pull/64/files#diff-4b7bcbde16b715290fbc84da32a3f1e2L14

Reply:
Thank you! Removed policy creation from the *New asset* menu list and added it to the Maven list.